.github/workflows/e2e.yaml

```
@@ -0,0 +1,37 @@
+name: E2E Tests
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
```

This should be equal to the one on https://github.com/vtex/shipping-manager/pull/92 I guess? Same goes for the file name

```suggestion
name: IO App Cypress

on:
  push:
    branches: [ "main" ]
  pull_request:
    types:
    - opened
    - synchronize
```

---

it does the same thing I guess, but I'll change for consistency

---

done